docker-compose.yml

```diff
@@ -25,9 +25,15 @@ services:
       context: ./src/backend
     environment:
       - ASPNETCORE_ENVIRONMENT=Development
-    restart: always
+      - ASPNETCORE_URLS=http://0.0.0.0:8080
+      - SERILOG__WRITETO__1__NAME=Seq
+      - SERILOG__WRITETO__1__ARGS__SERVERURL=http://seq:5341
     ports:
       - "8081:8080"
+    volumes:
+      - ${APPDATA}/Microsoft/UserSecrets:/root/.microsoft/usersecrets:ro
```

Comment:
This maps the developer's user secrets so they can be used inside the container, keeping secrets out of the source tree.